deploy/README.md

```diff
@@ -0,0 +1,62 @@
+# Server deployment
+
+> [!WARNING]
+> This is a work in progress and not production-ready yet.
+
+## Context
+
+This directory contains the necessary files to deploy a multi-omics digital objects (MODO) server.
+
+The modo-server is meant to provide remote access to the MODOs. Currently, it can:
+
+* [x] list available MODOstheir
+* [x] return their metadataThe
+* [ ] stream CRAM slices CRAM using htsget
+* [ ] expose a MODO directly as a client-accessible S3 bucket / folder
+* [ ] manage authentication and access control
+
+The MODOs are stored in an s3 (minio) bucket, and an htsget server is deployed alongside the modo-server to handle slicing and streaming of CRAM files. A REST API is exposed to the client to interact with the remote MODOs.
+
+
+```mermaid
+
+%%{init: {'theme': 'default'}}%%
+flowchart TB
+  Vdatas3{{/data/s3}} x-. /bitnami/minio/data .-x minio
+  htsget --> minio
+  modoserver[modo-server] --> minio
+  modoserver --> htsget
+  minio -.- modonetwork[/modo-network/]
+  htsget -.- modonetwork
+  modoserver -.- modonetwork
+  modoclient[modo-client] --> modoserver
+
+  classDef volumes fill:#fdfae4,stroke:#867a22
+  class Vdatas3 volumes
+  classDef nets fill:#fbfff7,stroke:#8bc34a
+  class modonetwork nets
+
+```
+
+## Setup
+
+> [!IMPORTANT]
+> The instructions below are meant for local testing.
+> Production use would require changing the default
+> credentials and use authentication.
+
+1. Start the server
+```sh
+cd deploy
+mkdir -p /data/s3 # Directory used to persist S3 data
```

Comment:
Solution 1 is indeed better practice, so we don't tie to a specific folder and the storage is managed by docker itself.

I have updated the compose file + docs to use a volume instead of `/data/s3` let me know if that solves your issue.

Reply:
Great! Thanks a lot! It works perfect now!